release-notes/2023-03/README.md

```diff
@@ -0,0 +1,214 @@
+# SystemLink Enterprise 2023-03 Release Notes
+
+The 2023-03 bundle for SystemLink Enterprise has been published to <https://niedge01.jfrog.io>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## New Features and Behavior changes
+
+- Support for Kubernetes 1.23
+
+- Support for both PostgreSQL 13 and 14.
+
+- Users can now export data tables to CSV
+
+- Product, results, and steps tables can be queried using the table query builder.
+
+- OpenTelemetry metrics are now available for the following services
+    - testmonitor
+    - dataframeservice
+    - Kafka
+
+- The endpoint to view claims for the current logged in user, `/oidc/userinfo`, has been removed and replaced with `oidc/user-info`.
+
+- Kafka UI is now available for debugging and monitoring
+
+- Kafka S3 sink connectors are automatically deleted if a data table is marked `endOfData`.
+
+- Deleted Kafka S3 sink connectors are automatically reestablished if connectors are manually deleted.
+
+- The default memory provided to Kafka topic operator has increased from 256MB to 512MB to increase the total number of tables that can be written to by the Data Frame Service.
+
+## Helm Chart Breaking Changes
+
+- `systemlink 0.11.81`
+    - Setting image pull secrets
+        - Image pull secret specification has been split into two separate global values.
+        - You must specifying the single required image pull secret in a new value `global.niImagePullSecret` and reference this value in `global.imagePullSecrets`.
+        - [View this configuration](https://github.com/ni/install-systemlink-enterprise/blob/2023-03/getting-started/templates/systemlink-values.yaml#L18)
+
+    - Enabling Kafka OpenTelemetry metrics
+        - Set `global.telemetry.enabled` flag to `true`.
+        - You are no longer required to also set options in `kafka.metricsConfigs`, `kafka.jmxOptions`, and `kafka.kafkaExporter` configurations.
+        - [View this configuration](https://github.com/ni/install-systemlink-enterprise/blob/2023-03/getting-started/templates/systemlink-values.yaml#L56)
+
+    - Enabling user claims to be returned in a web browser.
+        - Set `webserver.oidc.enableUserInfo` to `true`.
+        - You should remove `offline_access` from `webserver.oidc.scope`.
+        - [View this configuration](https://github.com/ni/install-systemlink-enterprise/blob/2023-03/getting-started/templates/systemlink-values.yaml#L110).
+
+    - Enabling Kafka UI
```

Comment:
This section is no longer needed after the following PR goes in:
https://dev.azure.com/ni/DevCentral/_git/Skyline/pullrequest/454125

Once we rebuild the release bundle and validate this I will update the template values file and we should be able to remove this section from the docs.

Reply:
Well, that's not exactly true. We will no longer need to specify the image registry override. We should add a blurb here about needing the specify the image pull secrets as outlined by the following PR:

https://github.com/ni/install-systemlink-enterprise/pull/116

Reply:
@cameronwaterman please check the wording for this section. A couple of questions that might have me change the wording as it stands now:
- Would this Kafka-ui image pull secret be the same as the pull secrets specified in other places?
- Is there any update to the secrets template that would need to be required, or would we simply reference the pull-secret value as we would in other places?

Reply:
We might want to be a bit more specific. Maybe the section header should read "Enabling Kafka UI for the Data Frame service Kafka deployment" (or something along those lines), as the Kafka cluster we deploy is specific to the DFS and deployed by the DFS.

And then the details would read "To enable Kafka UI for the Data Frame service Kafka deployment, specify `dataframeservice.kafka-ui.imagePullSecrets`. We might want to also link to the [use of the YAML alias in the template values](https://github.com/ni/install-systemlink-enterprise/blob/main/getting-started/templates/systemlink-values.yaml#L613) to drive home the point that this value can and should be single sourced.

Yes the image pull secret is the same and there is no required updates to the secrets template.